instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -117,6 +117,7 @@ def process_request(self, request):
             )
             for key, value in attributes.items():
                 span.set_attribute(key, value)
+            span.set_attribute("http.path", request.path)
```

Comment:
This should be `http.target` according to the spec.     https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes

Reply:
Sound good.

Reply:
Isn't `http.target` already being populated in `collect_request_attributes`?

Reply:
Good point @lzchen!  I will remove the setting of the attribute from the django middleware.